src/components/FormSelectList.vue

```diff
@@ -236,7 +244,7 @@ export default {
         return true;
       } catch (err) {
         /* Ignore error */
-        console.error(err);
+        console.warn(err);
```

Comment:
any reason the switch from `.error` to `.warn` ?

Reply:
Change to error is a change of the expected behavior. Take in count that QA reviews the error messages, and this could be a new one not expected.

Reply:
Note that: Originally (previous to the FOUR-6721 performance improvements) this was `console.warn` not `console.error`